package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "@chamaeleonidae/chmln",
+  "version": "1.0.0-alpha.2",
```

Comment:
There's no automatic versioning or changelog generator, for now, we can add it in the next few days in new PR's as we improve the npm publish flow and get more people involved in the open-source process of this. Also, this is `alpha.2` and can be upgraded/changed lots of times before we really have a `1.0.0` final, and this can be fone right before we deploy the new installation page that is on backlog now